proto/doc.go

```diff
@@ -1,3 +1,5 @@
 // Package proto contains protocol buffers that are exchanged between the client
 // and server. Note, some of these types have additional helper methods.
 package proto
+
+//go:generate protoc --go_out=. --go_opt=module=github.com/google/go-tpm-tools/proto tpm.proto
```

Comment:
Not important now: do we want to keep it *.proto, in case we add protos outside of the tpm.proto file in the future?

Reply:
I was initially going to do this, but `go:generate` doesn't expand wildcards